apps/examples/araplayer/PlaylistInfo.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include "Enumerator.h"
+#include "HASType.h"
+
+struct play_list_info_s {
```

Comment:
please add description.

Reply:
we will add a description of each structure and enum to the next commit.